### DISCUSSION TITLE
So is this the best fork to switch to? Automatic1111 hasn't made a commit in 3 months.

### DISCUSSION THREAD

**Discussion by CalculonPrime** [vladmandic/sdnext#3524] [General]
In a fast changing area like AI, I can't stick with a project that never gets updates.

But how is the stability of this repo?

**Comment by vladmandic** [comment]
i cant say whats best for you - its your choice.
perhaps you can jump in on discord and see what community is saying, its ~3k users live.

**Comment by tampadesignr** [comment]
> In a fast changing area like AI, I can't stick with a project that never gets updates.
> 
> But how is the stability of this repo?

this guy has been working and answering people and pushing the limits of a gui for ai
give the gui a test run

as far as stability? depends on a number of factors like hardware, software etc..
but this gui is trying everything out so some stuff might not work for your setup but its worth a look
